src/main/java/com/textadventure/Lexer.java

```diff
@@ -41,10 +38,9 @@ private TokenType(String pattern) {
 			} else if (matcher.group(TokenType.DIRECTION.name()) != null) {
 				tokens.add(new Token(TokenType.DIRECTION, matcher.group(TokenType.DIRECTION.name())));
 				continue;
-			} else if (matcher.group(TokenType.WHITESPACE.name()) != null)
-				continue;
+			}
+		}
 		}
```

Comment:
This line and above line indicate something wrongly indented. Check your loop start and end to see if you forgot to indent something.